utils/appliance.py

```diff
@@ -1146,23 +1200,45 @@ def is_web_ui_running(self, unsure=False):
         else:
             return unsure
 
-    def restart_evm_service(self, rude=False, log_callback=None):
+    def restart_evm_service(self, log_callback=None):
```

Comment:
removing `rude` is a backward-incompatible change, and doesn't appear to be necessary.